packages/desktop-client/src/components/schedules/EditSchedule.js

```diff
@@ -624,7 +618,6 @@ export default function ScheduleDetails({ modalProps, actions, id }) {
             style={{
               width: 350,
               textAlign: 'right',
-              color: colors.n4,
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏wrong color; also the text gets broken up incorrectly (when comparing against v23.8.1)

Before:
<img width="328" alt="Screenshot 2023-08-12 at 09 29 10" src="https://github.com/actualbudget/actual/assets/886567/83d81adb-6a52-4d0b-8c5e-83aeac29032f">


After:
<img width="343" alt="Screenshot 2023-08-12 at 09 29 14" src="https://github.com/actualbudget/actual/assets/886567/c9f26c53-faed-4a7b-acfc-5a373fb7ab35">

Reply:
This formatting is an issue with the current master and is not a result of anything within this PR.

I have fixed the colors.